probe_scraper/scrapers/moz_central_scraper.py

```diff
@@ -62,39 +63,67 @@ def load_tags(channel):
     return data
 
 
-def extract_tag_data(tag_data, channel):
+def extract_major_version(version_str):
```

Comment:
Can we add test coverage for this bunch of functions?

Reply:
I did! Check `test_extract_major_version`